src/vivarium_public_health/risks/data_transformation.py

```diff
@@ -90,3 +90,22 @@ def compute_paf(g):
     paf = paf.replace(-np.inf, 0)  # Rows with zero exposure.
 
     return paf
+
+
+def split_risk_from_type(full_risk: str):
+    """Expecting risk to specified as type.name (where type is singular).
+    Splitting out type and name."""
+    split = full_risk.split('.')
+    if len(split) != 2:
+        raise ValueError(f"You must specify the risk as <risk_type.risk_name>. You specified {full_risk}.")
+    return split[0], split[1]
+
+
+def split_target_from_type_entity(full_target: str):
+    """Expecting affected entity to be specified as type.name.target (where type is singular).
+    Splitting out type, name, and target. """
+    split = full_target.split('.')
+    if len(split) != 3:
+        raise ValueError(f"You must specify the target as <affected_entity_type.affected_entity_name.target_pipeline>. "
```

Comment:
Same note as above.  Also, probably specified as `affected_entity_type.affected_entity_name.affected_measure`.  Both the name and the measure make up the pipeline so this is kind of ambiguous.